src/components/form-elements/Textarea.jsx

```diff
@@ -5,25 +5,81 @@ import classnames from 'classnames';
 
 type TextareaSizeType = 'short' | 'normal' | 'tall' | 'xtall';
 
+type TextareaColorType = 'default' | 'white';
+
 export const SIZE = {
   SHORT: 'short',
   NORMAL: 'normal',
   TALL: 'tall',
   XTALL: 'xtall',
 };
 
+export const TEXTAREA_COLOR = {
+  DEFAULT: 'default',
```

Comment:
isn't the no-option indicate the default behaviour (and look)?

Reply:
We need it for export to sketch.

Reply:
oh I see